README.md

```diff
@@ -219,7 +225,7 @@ A [sample project](sample) for Android and iOS is available.
 ## Talk
 
 The idea and more background about this library is covered in this
-[public talk](https://ralf-wondratschek.com/presentation/extending-kotlin-inject.html).
+[public talk](https://ralf-wondratschek.com/presentation/extending-kotlin-inject-nyc.html).
```

Comment:
That was fast 🙃 I didn't mean to update the link yet without the recording.

Reply:
atm it `404`s so figured made sense to point to the latest talk. happy to revert or point to a different link if you prefer.

Reply:
I totally missed that 🤦‍♂️